Automattic/vip-cli
# Add debug info on error

## Description

Adds additional debug information that can be useful in support requests, including VIP-CLI, Node, and OS versions. It places this debug info below the error message so that someone who begins their text selection on the error message can easily extend their selection down to include it.

```sh
> vip ice-cream
Error:  `ice-cream` is not a valid subcommand. See `vip --help`
Debug:  VIP-CLI v2.0.7, Node v14.16.1, darwin 20.4.0
```

This also introduces more consistency to error output by always prepending "Error:" and always exiting with `1`.

## Steps to Test

1. Check out PR.
1. Run `npm run build`
1. Generate an error, e.g., `node ./dist/bin/vip ice-cream`
1. Inspect error output
